src/ripple/app/main/LoadManager.cpp

```diff
@@ -129,23 +129,32 @@ void LoadManager::run ()
             auto const reportingIntervalSeconds = 10s;
             if (armed && (timeSpentDeadlocked >= reportingIntervalSeconds))
             {
+                constexpr auto deadlockTimeLimit = 90s;
+
                 // Report the deadlocked condition every 10 seconds
                 if ((timeSpentDeadlocked % reportingIntervalSeconds) == 0s)
                 {
-                    JLOG(journal_.warn())
-                        << "Server stalled for "
-                        << timeSpentDeadlocked.count() << " seconds.";
+                    if (timeSpentDeadlocked < deadlockTimeLimit)
+                    {
+                        JLOG(journal_.warn())
+                            << "Server stalled for "
+                            << timeSpentDeadlocked.count() << " seconds.";
+                    }
+                    else
+                    {
+                        JLOG(journal_.fatal())
+                            << "Deadlock detected. Deadlocked time: "
+                            << timeSpentDeadlocked.count()
+                            << "s JobQueue Overloaded: "
```

Comment:
I think what @miguelportilla had in mind was, if the job queue is overloaded then write the job queue status json at fatal level.  Maybe something like this:
```
        if (app_.getJobQueue().isOverloaded())
        {
            JLOG(journal_.fatal()) << app_.getJobQueue().getJson(0);
        }
```

Reply:
@scottschurr Indeed.

Reply:
@seelabs, what do you think about revising the logging in this way?

Reply:
@miguelportilla @scottschurr Fixed in `b9b524917  [FOLD] Log more info about the job queue when overloaded`